lib/spack/spack/test/packaging.py

```diff
@@ -199,6 +200,7 @@ def test_needs_relocation():
     assert needs_binary_relocation("application", "x-mach-binary")
 
 
+@pytest.mark.not_on_windows("macos use not intended for Windows")
```

Comment:
I'm confused by this because it's also not intended for Linux: can you rephrase in terms of whatever part of this doesn't work on Windows? Could be something like "dylib manipulation logic in X does not run on Windows" (where X is whatever function in here fails on Windows).

Reply:
Okay. I just changed the pytest descriptions to be clearer.